changelog.md

```diff
@@ -2,6 +2,15 @@
 
 ## NEXT
 
+- Added `cai_causal_graph.time_series_causal_graph.TimeSeriesNode` class to extend the `cai_causal_graph.graph_components.Node` class to time series representations of causal graphs. The main additions are:
+  - `variable_name`: the variable name of the time series node. For example, if the time series node is a lagged version of the variable `x`, then `variable_name` would be `x`.
+  - `time_lag` attribute that represents the time lag of the time series node.
+- Added `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` class to deal with time series representations of causal graphs. The main differences from the `cai_causal_graph.causal_graph.CausalGraph` class is that the `TimeSeriesCausalGraph` is aware of time. Moreover, the `TimeSeriesCausalGraph` class has three new representations of the causal graph:
+  - minimal graph (obtained with the method `get_minimal_graph`) defined as the graph with the minimal number of edges that is equivalent to the original graph. In other words, it is a graph that has no edges whose destination is not time delta 0.
```

Comment:
Also, you can format `get_minimal_graph` as `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.get_minimal_graph`